scripts/packaging/prepare-containers.ts

```diff
@@ -0,0 +1,98 @@
+///////////////////////////////////////// WARNING /////////////////////////////////////////
+//                                                                                       //
+//                 Do NOT run these scripts outside a Docker container!                  //
```

Comment:
Can we check this in this from with nodejs? I did a quick google and found https://www.npmjs.com/package/is-docker.

Reply:
I don't really think it's necessary. Nobody's going to run those scripts unless they're targeted in `package.json`.

Reply:
But what is the disadvantage of doing it as I proposed?

Reply:
I'm a bit worried about false positives but I can try it.

Reply:
I had to take the code from the library because it uses ESM modules which don't work with our setup. But it looks like it's working. At least for my setup :sweat_smile:

Reply:
> I had to take the code from the library because it uses ESM modules which don't work with our setup

You can just use major version 2. See: https://github.com/sindresorhus/is-docker/releases/tag/v3.0.0

Reply:
I'm really not going to spend more time on this, it's a useless check anyway :smile: